### PR TITLE
User Login Action

### PR DESCRIPTION
**Pull request by amimihtun-scm**
<< Changes >>
-Validation
-Login Action
-Logout Action

<< Impact Range >>
-

<< Related APIs >>
-

<< Test Items >>
-User Login Page
-Post List Page

<< To Do >>
_

<< Points to Check Thoroughly >>
RD Document file need to check 2 timing

<< Note >>
-Login with seeded email from database and password (bulletinboard) 

<< Captures >>
image
image
image
image
image
image
image

### CODE REVIEW

File: bulletinboard/app/Http/Requests/UserLoginRequest.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+        return [
+            'email' => 'required|email:rfc,dns|max:50|exists:users,email',
```
Comment (Mya-Myintzu-Thin) [inline bulletinboard/app/Http/Requests/UserLoginRequest.php:27]: email format hmr rfc,dns ma lo yin ma htae pr nae

File: bulletinboard/app/Http/Requests/UserLoginRequest.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            // 'password' => 'required|min:6|max:20|regex:/^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{6,}$/|exists:users,password',
```
Comment (Mya-Myintzu-Thin) [inline bulletinboard/app/Http/Requests/UserLoginRequest.php:28]: comment code delete pay prr

File: bulletinboard/app/Services/Post/PostServices.php [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    /**
+     * To submit Post login 
+     * @param $request
+     * @return View Posts 
+     */
+    public function login($request)
+    {
+        
+    }
+}
```
Comment (Mya-Myintzu-Thin) [inline bulletinboard/app/Services/Post/PostServices.php:78]: Doc comment tway wrong nay pr tl

File: bulletinboard/routes/web.php [hunk 4]
```diff
@@ -14,5 +16,63 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('layouts.index');
+});
+
+Route::get('/login', [UserLoginController::class, "index"])->name("login");
+Route::post('/user/login', [UserLoginController::class, "login"])->name("user.login");
+Route::get('/logout', [UserLoginController::class, "logout"])->name("user.logout");
```
Comment (PyaeHan) [inline bulletinboard/routes/web.php:24]: Use all with single quotes.
e.g. Route::get('/login', [UserLoginController::class, 'index'])->name('login');

**Pull request merged**